src/main/java/net/minecraftforge/event/entity/player/AdvancementEvent.java

```diff
@@ -17,6 +18,7 @@
  * This event does not have a result. {@link HasResult}<br>
  * <br>
  * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ * @Deprecated Use {@link net.minecraftforge.event.entity.player.AdvancementEvent.AdvancementEarnEvent} and {@link net.minecraftforge.event.entity.player.AdvancementEvent.AdvancementProgressEvent} instead
  */
 public class AdvancementEvent extends PlayerEvent
```

Comment:
The deprecated tag in the javadoc should be `@deprecated` in all lowercase. The annotation should also be present on the event, though not deprecating the event for removal:
```suggestion
 * @deprecated Use {@link net.minecraftforge.event.entity.player.AdvancementEvent.AdvancementEarnEvent} and {@link net.minecraftforge.event.entity.player.AdvancementEvent.AdvancementProgressEvent} instead.
 */
@Deprecated(since = "1.19.2")
public class AdvancementEvent extends PlayerEvent
```